src/api_manager/auth/lib/auth_jwt_validator.cc

```diff
@@ -612,7 +652,36 @@ bool JwtValidatorImpl::ExtractPubkeyFromJwk(const grpc_json *jkey) {
   return true;
 }
 
-grpc_jwt_verifier_status JwtValidatorImpl::VerifyRsSignature(const char *pkey,
+bool JwtValidatorImpl::ExtractPubkeyFromJwkEC(const grpc_json *jkey) {
+  if (eck_ != nullptr) {
+    EC_KEY_free(eck_);
+  }
```

Comment:
According to documentation, EC_KEY_free frees all the data owned by key and key itself.